components/src/histogram.tsx

```diff
@@ -19,18 +19,10 @@
  *
  */
 
-import iobioCharts from 'iobio-charts';
-import { useEffect } from 'react';
+import 'iobio-charts';
 
-const { createHistogram } = iobioCharts;
-
-function Histogram(props: { 'broker-key': string }) {
-	useEffect(() => {
-		createHistogram();
-	}, []);
-
-	// return <iobio-histogram />;
-	return <div broker-key={props['broker-key']}></div>;
+function IobioHistogram({ brokerKey, color }: { brokerKey: string; color?: string }) {
+	return <iobio-histogram broker-key={brokerKey} style={{ '--iobio-data-color': color }} />;
```

Comment:
potentially 'better' solution than style prop? but for now this is locally scoped

Reply:
color will be added to other components in later iobio-charts versions